### PR TITLE
Change storage system to .txt

### PR DESCRIPTION
**Pull request by NigelYeoTW**
Will store into ./spendings.txt and ./incomes.txt with pipeline seperators

### CODE REVIEW

File: src/main/java/seedu/storage/IncomeListStorage.java [hunk 1]
```diff
@@ -1,34 +1,70 @@
...
+                }
+                Income nextEntry = new Income(Double.parseDouble(entryData[LOAD_AMOUNT_INDEX]),
+                        entryData[LOAD_DESCRIPTION_INDEX], date, entryData[LOAD_TAG_INDEX],
+                        RecurrenceFrequency.valueOf(entryData[LOAD_RECURRENCE_INDEX]),
+                        lastRecurred, Integer.parseInt(entryData[LOAD_DAY_OF_RECURRENCE_INDEX]));
+                Storage.incomes.add(nextEntry);
+            }
         } catch (IOException e) {
-            Ui.printWithTab(e.getMessage());
+            Ui.printWithTab("An error has occurred when saving file!");
```
[the remark this answers is not in the document]
Comment (NigelYeoTW) [reply src/main/java/seedu/storage/IncomeListStorage.java]: new users will now not see the error message, added guard clause for early return.
error fixed.

File: src/main/java/seedu/storage/SpendingListStorage.java [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
+import static seedu.classes.Constants.LOAD_AMOUNT_INDEX;
+import static seedu.classes.Constants.LOAD_DATE_INDEX;
+import static seedu.classes.Constants.LOAD_DAY_OF_RECURRENCE_INDEX;
+import static seedu.classes.Constants.LOAD_DESCRIPTION_INDEX;
+import static seedu.classes.Constants.LOAD_LAST_RECURRED_INDEX;
+import static seedu.classes.Constants.LOAD_RECURRENCE_INDEX;
+import static seedu.classes.Constants.LOAD_TAG_INDEX;
+import static seedu.classes.Constants.NO_RECURRENCE;
 
 public class SpendingListStorage {
```
[the remark this answers is not in the document]
Comment (NigelYeoTW) [reply src/main/java/seedu/storage/SpendingListStorage.java:28]: budget now saved in first line of spending file, will always read the first line first before inputting spending data

File: src/main/java/seedu/storage/IncomeListStorage.java [hunk 3]
```diff
@@ -1,34 +1,70 @@
...
+                        CSV_SEPARATOR + income.getDate() + CSV_SEPARATOR + income.getTag() + CSV_SEPARATOR +
+                        income.getRecurrenceFrequency() + CSV_SEPARATOR + income.getLastRecurrence() +
+                        CSV_SEPARATOR + income.getDayOfRecurrence();
+                bw.write(incomeEntry);
+                bw.newLine();
+            }
+            bw.flush();
+            bw.close();
+        } catch (IOException e){
+            Ui.printWithTab("An error has occurred when saving file!");
```
[the remark this answers is not in the document]
Comment (NigelYeoTW) [reply src/main/java/seedu/storage/IncomeListStorage.java]: removed and specified issue

**Comment by NigelYeoTW** [comment]
May need to refactor the code, method seems a bit too long. But general implementation in up please help check for errors. ty.

**Comment by TPH777** [comment]
lgtm!

**Pull request merged**